app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -132,6 +132,7 @@ type CustomizableAreaValue @doc(description: "CustomizableAreaValue defines the
     price_type: PriceTypeEnum @doc(description: "FIXED, PERCENT, or DYNAMIC.")
     sku: String @doc(description: "The Stock Keeping Unit for this option.")
     max_characters: Int @doc(description: "The maximum number of characters that can be entered for this customizable option.")
+    id_v2: String @doc(description: "Base64 string, that encodes details for each option.") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\CustomizableEnteredOptionValueIdV2")
```

Comment:
@keharper Please review the schema descriptions.

Reply:
@mmezhensky Change the descriptions of `id_v2` to "A base64 string that encodes option details."

Reply:
IDs should be opaque to a client - let's avoid putting anything about base64 encoding into the docs for ID fields - it's an implementation detail.

It can be an inline comment in the `.graphqls` file with the `#` character as a hint to developers working on the implementation, though.

Reply:
@keharper, @DrewML the description were updated. The Base64 encoding is mentioning as field comment.